assets/component-rte.css

```
@@ -1,3 +1,9 @@
+.rte:after {
+  clear: both;
+  content: "";
```

For consistency with other references:
```suggestion
  content: '';
```